_episodes/02-project-planning.md

```diff
@@ -1,110 +1,143 @@
+facility. Take a look at the following example submission spreadsheet.
+
+[Sample submission sheet](../files/sample_submission.txt)
+
+Download the file using right-click (PC)/command-click (Mac). This is a tab-delimited text file. Try opening it
+with Excel or another spreadsheet program.
+
+> ## Exercise
+>
+> 1. What are some errors you can spot in the data? Typos, missing data, inconsistencies?
+> 2. What improvements could be made to the choices in naming?
+> 3. What are some errors in the spreadsheet that would be difficult to spot? Is there anyway you can test this?
+>
+> > ## Solution
+> > Errors:
+> > - Sequential order of well_position changes
+> > - Format of client_sample_id changes and cannot have spaces, slashes, non-standard ASCII characters
+> > - Capitalization of the replicate column changes
+> > - Volume and concentration column headers have unusual (not allowed) characters
+> > - Volume, concentration, and RIN column decimal accuracy changes
+> > - The prep_date and ship_date formats are different, prep_date has multiple formats 
+> > - Are there others not mentioned?
+> >
+> > Improvements in naming
+> > - Shorten client_sample_id names, and maybe just call them "names"
+> >   - For example: "wt" for "wild-type". Also, they are all "1hr" so that is superfluous information
+> > - The prep_date and ship_date might not be needed
+> > - Use "microliters" for "Volume (Â¬ÂµL)" etc. 
+> >
+> > Errors hard to spot:
+> > - No space between "wild" and "type", repeated barcode numbers, missing data, duplicate names
+> > - Find by sorting, or counting
+> >
+> {: .solution}
+{: .challenge}
+
+# Retrieving samples from the facility
+
+When the data come back from the sequencing facility, you will receive some documentation (metadata) as well as
+the sequence files themselves. Download and examine the following example file - here provided as a text file and
+Excel file:
+
+- [Sequencing results - text](../files/sequencing_results_metadata.txt)
+- [Sequencing results - Excel](../files/sequencing_results_metadata.xls)
+
+> ## Exercise
+>
+> 1. How are these samples organized?
+> 2. If you wanted to relate file names to the sample names submitted above (e.g. wild type...) could you do so?
+> 3. What do the \_R1/\_R2 extensions mean in the file names?
+> 4. What does the '.gz' extension on the filenames indicate?
+> 5. What is the total file size - what challenges in downloading and sharing these data might exist?  
+>
+> > ## Solution
+> >
+> > - Samples are organized by sample_id
+> > - To relate filenames use the sample_id, and do a VLOOKUP on submission sheet
+> > - The \_R1/\_R2 extensions mean "Read 1 and "Read 2 of each sample
+> > - The '.gz' extension means it is a compressed "gzip" type format to save disk space
+> > - The size of all the files combined is 1113.60 gb (over a terabyte!). To move files this large will
+> >   require using checksums to validate file integrity after transfers, and a fast transfer protocol

```

Comment:
Do we discuss checksums in this lesson? If not, I recommend adding something like "this is beyond the scope of this lesson".